calico/getting-started/kubernetes/k8s-single-node.mdx

```diff
@@ -141,20 +144,24 @@ The geeky details of what you get:
    node/<your-hostname> untainted
    ```
 
-1. Confirm that you now have a node in your cluster with the following command.
+3. Monitor the deployment by running the following command:
```

Comment:
I'm trying to change the verification steps to using tigerastatus. But now I'm viewing this and thinking the confirmation of a node is different. Should I revert?